feat(profiling): add browser profiling docs (#7468)
getsentry/sentry-docs: src/wizard/javascript/profiling-onboarding/javascript/2.configure-document-policy.md

---
name: Browser (JS Self Profiling)
doc_link: https://docs.sentry.io/platforms/javascript/profiling/
support_level: beta
type: framework
---

> #### Add Document-Policy:js-profiling header

Reviewer: ```suggestion
## Add Document-Policy: js-profiling header
```

> In order to enable the profiler to start, the document response header has to include a header key `Document-Policy` with value of `js-profiling`.

Reviewer: ```suggestion
In order for the profiler to start, the document response header has to include a `Document-Policy` header key with the `js-profiling` value.
```

> Depending on your server, how you do this will vary. If you are using a server like express, you can do this like via the [response.set](https://expressjs.com/en/4x/api.html#res.set) function.

Reviewer: ```suggestion
How you do this will depend on your server. If you're using a server like express, you'll be able to use the [response.set](https://expressjs.com/en/4x/api.html#res.set) function.
```
